scripts/bumpVersion.js

```diff
@@ -7,36 +8,9 @@ const versionBump = process.argv[2] || "patch"
 const fileData = fs.readFileSync(filePath)
 const lernaData = JSON.parse(fileData)
 
-// Get current version and split into major, minor, patch, and alpha components
 const currentVersion = lernaData.version
-const [versionWithoutPrerelease, alpha] = currentVersion.split("-")
-const [major, minor, patch] = versionWithoutPrerelease.split(".").map(Number)
 
-// Calculate new version based on specified version bump
-let newVersion = currentVersion
-switch (versionBump) {
-  case "major":
-    newVersion = `${major + 1}.0.0`
-    break
-  case "minor":
-    newVersion = `${major}.${minor + 1}.0`
-    break
-  case "patch":
-    newVersion = `${major}.${minor}.${patch + 1}`
-    break
-  case "alpha":
-    let newAlphaVersion = 0
-    if (alpha) {
-      const [_, alphaVersion] = alpha.split(".")
-      newAlphaVersion = +alphaVersion + 1
-    }
-    newVersion = `${versionWithoutPrerelease}-alpha.${newAlphaVersion}`
-
-    break
-  default:
-    console.error(`Invalid version bump '${versionBump}' specified.`)
-    process.exit(1)
-}
+const newVersion = semver.inc(currentVersion, versionBump, "alpha")
```

Comment:
Is the `alpha` ignored if the `versionBump` is not `prerelease`, or how does that work?

Reply:
Yes, it's quite confusing, but it works like this:
```
Prerelease Identifiers

The method .inc takes an additional identifier string argument that will append the value of the string as a prerelease identifier:

semver.inc('1.2.3', 'prerelease', 'beta')
// '1.2.4-beta.0'

command-line example:
```

It's only used for prereleases.

Reply:
That's not clear at all lol, but good that it works!